src/node_jsvmapi.h

```diff
@@ -110,6 +110,7 @@ NODE_EXTERN napi_status napi_create_symbol(napi_env e,
 NODE_EXTERN napi_status napi_create_function(napi_env e,
                                              napi_callback cb,
                                              void* data,
+                                             napi_propertyname name,
```

Comment:
That's true and matches how ```napi_define_class``` names the constructor function. Inside napi, both ```napi_create_function``` and ```napi_define_class``` need to construct ```napi_value``` for the name string but this is an implementation detail. I'll change this to const char* as it's more consistent.